packages/shell-chrome/src/devtools/devtools.js

```diff
@@ -50,6 +51,9 @@ export default function devtools() {
         settingsPanelEnabled: process.env.NODE_ENV !== 'production',
         settingsPanelOpen: false,
 
+        tabsEnabled: process.env.NODE_ENV !== 'production',
```

Comment:
I mentioned this to @KevinBatdorf on his PR (https://github.com/alpine-collective/alpinejs-devtools/pull/136#discussion_r546734460), now that we've got Edge.js we can if/else in edge templates instead of setting variables & hiding with Alpine, what do you think?

Reply:
Yeah that's a good shout. I'll add that in.

Reply:
Can this line be deleted now since it was added elsewhere?